src/WebApp/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "noImplicitAny": false,
+    "noEmitOnError": true,
+    "removeComments": false,
+    "sourceMap": true,
+    "target": "es5"
```

Comment:
I thought we were targeting ES6? I do not know where I picked that up though.

Reply:
If that was the case then there would be no point in adding TypeScript.
Just adding this instead of babel basically.

Reply:
I mean TypeScript does a lot more than just babel. Not sure if I am pinging the right person here, but @mattjbrent you do a lot of frontend stuff right? ES5, ES6 or TypeScript?

Reply:
Just browsing past... TS will give you some type safety in the front-end (at build-time, not run-time) and good Intellisense from VS Code. It's pretty useful, with great editor support. In terms of the build target though, it probably comes down to which browsers you're planning to support. `es5` might be safer.

Reply:
Let's use es5 for the built JavaScript.